back/node_watcher/src/util/blockIndex.ts

```diff
@@ -0,0 +1,30 @@
+import { BlockIndexCreateInput, prisma } from '../generated/prisma-client'
+
+export function createBlockIndex(identifier: string, startFrom: number, index: number) {
```

Comment:
I guess all these functions here could be used directly.
There is a stray `a` function and `updateBlockIndex` that is empty.

Reply:
removed